src/Core/Checkout/Test/Customer/SalesChannel/AddWishlistProductRouteTest.php

```diff
@@ -85,6 +86,12 @@ protected function setUp(): void
     public function testAddProductShouldReturnSuccess(): void
     {
         $productData = $this->createProduct($this->context);
+        $dispatcher = $this->getContainer()->get('event_dispatcher');
+
+        $listener = static function (WishlistProductAddedEvent $event) use ($productData): void {
+            static::assertSame($productData[0], $event->getProductId());
+        };
```

Comment:
```suggestion
        $eventWasThrown = false;
        $listener = static function (WishlistProductAddedEvent $event) use ($productData, &$eventWasThrown): void {
            static::assertSame($productData[0], $event->getProductId());
            $eventWasThrown = true;
        };
```

and after you do the request do a `static::assertTrue($eventWasThrown)`

otherwise your test would still be green if the event would not be thrown at all.

Same is true for the test in the `Remove` route

Reply:
@keulinho Thanks for advice. This is my first PR, therefore I have to implement your standards.